ChangeLog.md

```diff
@@ -1,5 +1,34 @@
 # Changelog for polysemy
 
+
+## 1.2.0.0 (TODO)
```

Comment:
Marked the date as TODO here since I figure we might want to add more stuff before we publish this to Hackage.